e2e/node/acp.test.ts

```diff
@@ -142,7 +142,7 @@ describe("An ACP Solid server", () => {
         ? latest_getPublicAccess
         : legacy_getPublicAccess;
     await expect(
-      getSolidDataset(sessionDataset, { fetch: session.fetch })
+      getSolidDataset(sessionDataset, )
```

Comment:
```suggestion
      getSolidDataset(sessionDataset)
```

Reply:
That was a typo, the session is supposed to be used here ^^,
Thanks for spotting it!

Reply:
Only half-spotted the issue then ^^